lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -705,10 +705,11 @@ private function Annotation()
                     }
                 }
             } elseif (isset($this->imports[$loweredAlias])) {
-                $found = true;
                 $name  = (false !== $pos)
                     ? $this->imports[$loweredAlias] . substr($name, $pos)
                     : $this->imports[$loweredAlias];
+                $name = ltrim($name,'\\');
```

Comment:
this is duplicated below, but we need it here to make $this->classExists work

Reply:
Please avoid overwriting the variable: wrap the previous expression, if possible

Reply:
did it differently, okay like this?